velox/common/base/SkewedPartitionBalancer.cpp

```diff
@@ -208,6 +246,13 @@ void SkewedPartitionRebalancer::calculatePartitionProcessedBytes() {
   for (auto partition = 0; partition < partitionCount_; ++partition) {
     totalPartitionRowCount += partitionRowCount_[partition];
   }
+  if (totalPartitionRowCount <= 0) {
```

Comment:
totalPartitionRowCount is uint so maybe put the check to be == 0

Reply:
this is debug log